pulsar-client-cpp/lib/ClientImpl.cc

```diff
@@ -64,30 +64,42 @@ const std::string generateRandomName() {
 }
 typedef boost::unique_lock<boost::mutex> Lock;
 
+static const ClientConfiguration detectTls(const std::string& serviceUrl,
+                                           const ClientConfiguration& clientConfiguration) {
+    ClientConfiguration conf(clientConfiguration);
+    std::string https("https");
+    std::string pulsarSsl("pulsar+ssl");
```

Comment:
```cpp
static const std::string https("https");
static const std::string pulsarSsl("pulsar+ssl"); 
```

Reply:
Changed. c++ is so damn slow to compile.